<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>365</x>
     <y>35</y>
     <width>278</width>
     <height>766</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" >
    <item>
     <widget class="QComboBox" name="stylesCombo" />
    </item>
    <item>
     <widget class="QMdiArea" name="mdiArea" >
      <property name="contextMenuPolicy" >
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <widget class="QWidget" name="subwindow" >
       <property name="windowTitle" >
        <string>Subwindow</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5" >
        <item row="0" column="0" >
         <widget class="QSplitter" name="splitter" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="groupBox" >
           <property name="title" >
            <string>GroupBox</string>
           </property>
           <property name="flat" >
            <bool>false</bool>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_3" >
            <item row="0" column="0" >
             <widget class="QPushButton" name="pushButton_4" >
              <property name="text" >
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QRadioButton" name="radioButton_5" >
              <property name="text" >
               <string>RadioButton</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QCheckBox" name="checkBox_3" >
              <property name="text" >
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QTabWidget" name="tabWidget" >
           <property name="currentIndex" >
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab" >
            <property name="geometry" >
             <rect>
              <x>0</x>
              <y>0</y>
              <width>294</width>
              <height>220</height>
             </rect>
            </property>
            <attribute name="title" >
             <string>Tab 1</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_4" >
             <item row="0" column="0" >
              <widget class="QTreeWidget" name="treeWidget" >
               <column>
                <property name="text" >
                 <string>FIrst Column</string>
                </property>
               </column>
               <item>
                <property name="text" >
                 <string>Foo</string>
                </property>
                <item>
                 <property name="text" >
                  <string>Bar</string>
                 </property>
                </item>
               </item>
               <item>
                <property name="text" >
                 <string>Another item</string>
                </property>
                <item>
                 <property name="text" >
                  <string>SubItem</string>
                 </property>
                 <item>
                  <property name="text" >
                   <string>SubSubItem</string>
                  </property>
                 </item>
                </item>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_13" >
            <property name="geometry" >
             <rect>
              <x>0</x>
              <y>0</y>
              <width>294</width>
              <height>220</height>
             </rect>
            </property>
            <attribute name="title" >
             <string>Page</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_14" >
            <property name="geometry" >
             <rect>
              <x>0</x>
              <y>0</y>
              <width>294</width>
              <height>220</height>
             </rect>
            </property>
            <attribute name="title" >
             <string>Page</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_2" >
            <property name="geometry" >
             <rect>
              <x>0</x>
              <y>0</y>
              <width>294</width>
              <height>220</height>
             </rect>
            </property>
            <attribute name="title" >
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOne_Menu" >
    <property name="title" >
     <string>One Menu</string>
    </property>
    <addaction name="actionOne_Entry" />
    <addaction name="separator" />
    <addaction name="actionAnother_Entry" />
   </widget>
   <widget class="QMenu" name="menuOne_More_Menu" >
    <property name="title" >
     <string>One More Menu</string>
    </property>
    <addaction name="actionFoobar" />
   </widget>
   <addaction name="menuOne_Menu" />
   <addaction name="menuOne_More_Menu" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>801</y>
     <width>785</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>22</y>
     <width>785</width>
     <height>13</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="smallToolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>35</y>
     <width>13</width>
     <height>766</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2" >
   <property name="geometry" >
    <rect>
     <x>649</x>
     <y>35</y>
     <width>136</width>
     <height>399</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Boxes and Horizontal Sliders</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>26</y>
      <width>136</width>
      <height>373</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLineEdit" name="lineEdit" />
       </item>
       <item>
        <widget class="QComboBox" name="comboBox" >
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="notEditableCombo" />
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QScrollBar" name="horizontalScrollBar" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider" >
         <property name="value" >
          <number>30</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition" >
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar" >
         <property name="value" >
          <number>50</number>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textVisible" >
          <bool>true</bool>
         </property>
         <property name="invertedAppearance" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3" >
   <property name="geometry" >
    <rect>
     <x>649</x>
     <y>440</y>
     <width>136</width>
     <height>361</height>
    </rect>
   </property>
   <property name="floating" >
    <bool>false</bool>
   </property>
   <property name="windowTitle" >
    <string>Grid View</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>26</y>
      <width>136</width>
      <height>335</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QTableWidget" name="tableWidget" >
       <property name="rowCount" >
        <number>5</number>
       </property>
       <property name="columnCount" >
        <number>3</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget" >
   <property name="geometry" >
    <rect>
     <x>13</x>
     <y>35</y>
     <width>346</width>
     <height>198</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="floating" >
    <bool>false</bool>
   </property>
   <property name="windowTitle" >
    <string>Radio &amp;&amp; Checks</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>26</y>
      <width>346</width>
      <height>172</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7" >
     <item>
      <widget class="QTabWidget" name="tabWidget_4" >
       <property name="tabPosition" >
        <enum>QTabWidget::South</enum>
       </property>
       <property name="tabShape" >
        <enum>QTabWidget::Rounded</enum>
       </property>
       <widget class="QWidget" name="tab_7" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>316</width>
          <height>119</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Tab 1</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6" >
         <item>
          <layout class="QGridLayout" name="gridLayout_2" >
           <item row="0" column="0" >
            <widget class="QRadioButton" name="radioButton" >
             <property name="text" >
              <string>Enabled</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
             <property name="autoExclusive" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QCheckBox" name="checkBox" >
             <property name="text" >
              <string>Enabled</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
             <property name="tristate" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QRadioButton" name="radioButton_2" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Disabled</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
             <property name="autoExclusive" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QCheckBox" name="checkBox_2" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Disabled</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QRadioButton" name="radioButton_3" >
             <property name="text" >
              <string>Enabled</string>
             </property>
             <property name="autoExclusive" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QCheckBox" name="tristateCheck" >
             <property name="text" >
              <string>Tristate</string>
             </property>
             <property name="checked" >
              <bool>false</bool>
             </property>
             <property name="tristate" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QRadioButton" name="radioButton_4" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Disabled</string>
             </property>
             <property name="autoExclusive" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QCheckBox" name="tristateDisable" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Tristate Disabled</string>
             </property>
             <property name="tristate" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_11" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>316</width>
          <height>119</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_12" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>316</width>
          <height>119</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_8" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>316</width>
          <height>119</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5" >
   <property name="geometry" >
    <rect>
     <x>13</x>
     <y>239</y>
     <width>346</width>
     <height>278</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Vertical Sliders</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>26</y>
      <width>346</width>
      <height>252</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <item>
      <widget class="QTabWidget" name="tabWidget_2" >
       <property name="tabPosition" >
        <enum>QTabWidget::West</enum>
       </property>
       <property name="tabShape" >
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>293</width>
          <height>222</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Tab 1</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" >
           <item>
            <widget class="QFrame" name="frame" >
             <property name="frameShape" >
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8" >
              <item>
               <widget class="QDial" name="dial" />
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QScrollBar" name="verticalScrollBar" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="verticalSlider" >
             <property name="value" >
              <number>50</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="tickPosition" >
              <enum>QSlider::TicksAbove</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar_2" >
             <property name="value" >
              <number>50</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
        <zorder></zorder>
       </widget>
       <widget class="QWidget" name="tab_10" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>293</width>
          <height>222</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_9" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>293</width>
          <height>222</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_4" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>293</width>
          <height>222</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4" >
   <property name="geometry" >
    <rect>
     <x>13</x>
     <y>523</y>
     <width>346</width>
     <height>278</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Buttons</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>26</y>
      <width>346</width>
      <height>252</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5" >
     <item>
      <widget class="QTabWidget" name="tabWidget_3" >
       <property name="tabPosition" >
        <enum>QTabWidget::East</enum>
       </property>
       <property name="currentIndex" >
        <number>3</number>
       </property>
       <widget class="QWidget" name="tab_5" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>293</width>
          <height>222</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_15" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>293</width>
          <height>222</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_16" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>293</width>
          <height>222</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_6" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>293</width>
          <height>222</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Tab 2</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <layout class="QGridLayout" name="gridLayout" >
           <item row="0" column="0" >
            <widget class="QPushButton" name="pushButton" >
             <property name="text" >
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QPushButton" name="pushButton_2" >
             <property name="text" >
              <string>Toggle</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QPushButton" name="disabledPushButton" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Disabled</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QPushButton" name="pushButton_3" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Toggle Disabled</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QPushButton" name="defaultPushButton" >
             <property name="text" >
              <string>Default</string>
             </property>
             <property name="default" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QToolButton" name="toolButton" >
             <property name="text" >
              <string>ToolButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOne_Entry" >
   <property name="text" >
    <string>One Entry</string>
   </property>
  </action>
  <action name="actionAnother_Entry" >
   <property name="text" >
    <string>Another Entry</string>
   </property>
  </action>
  <action name="actionFoobar" >
   <property name="text" >
    <string>Foobar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
